### PR TITLE
migrate production data to staging - improvements

### PR DESCRIPTION
**Pull request by george43g**
This PR is regarding #3162 - to enable better management of backup data.

### CODE REVIEW

File: apps/backend-functions/src/pubsub/schedule-daily.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { pubsub } from 'firebase-functions';
+import { freeze } from '../backup';
+
+export default pubsub.schedule('0 0 * * *').onRun(context => {
+  // const { eventId, eventType, params, resource, timestamp, auth, authType } = context;
```
Comment (GrandSchtroumpf) [inline apps/backend-functions/src/pubsub/schedule-daily.ts:5]: Removeif you're not using.

File: apps/backend-functions/src/pubsub/schedule-daily.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { pubsub } from 'firebase-functions';
+import { freeze } from '../backup';
+
+export default pubsub.schedule('0 0 * * *').onRun(context => {
```
Comment (GrandSchtroumpf) [inline apps/backend-functions/src/pubsub/schedule-daily.ts:4]: It would make more sense for me to use a cron in the CI, so it happens only on prod

**Comment by fritzschoff** [comment]
@george43g is this PR ready for merging?

**Comment by phpgeekfr** [comment]
@george43g can we close ?

**Pull request closed without merging**